common/src/main/java/org/opensearch/ml/common/MLModel.java

```diff
@@ -482,6 +486,7 @@ public static MLModel parse(XContentParser parser, String algorithmName) throws
                 case USER:
                     user = User.parse(parser);
                     break;
+                case MODEL_TASK_TYPE_FIELD:
```

Comment:
I didn't tell you to add case here. 

What I tried to tell you is: [in this line of code](https://github.com/opensearch-project/ml-commons/blob/2d401bc0504ca09bd9d729c3c4d06dad08e86e14/ml-algorithms/src/main/java/org/opensearch/ml/engine/ModelHelper.java#L101), we expect `model_task_type` to be present in the config file for the pre-trained models. Can also add another line to expect `function_name`? 

Like it will be fine if it gets any of these two `model_task_type` or `function_name`

Reply:
Oh you meant the other way around? Let me make that change